app/javascript/components/pages/UnprotectedShow.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import UnprotectedShow from "./UnprotectedShow"
+
+describe("<UnprotectedShow />", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div")
+    render(<UnprotectedShow />, div)
+  })
+})
```

Comment:
Just a reminder that you will need 60% of you code covered by tests, so think about adding some more. :)